corehq/apps/smsbillables/models.py

```diff
@@ -417,6 +417,12 @@ def _get_twilio_client():
                 'receiving',
             ]:
                 raise RetryBillableTaskException
+            if twilio_message.price is None:
+                log_smsbillables_error(
+                    "price is None: backend_message_id=%s, status=%s"
+                    % (twilio_message.backend_message_id, twilio_message.status)
```

Comment:
@nickpell just realized this should be the local backend_message_id, not twilio_message.backend_message_id